doc/auth.md

```diff
@@ -0,0 +1,78 @@
+# Enable Authentication
```

Comment:
It would be better to add the link of this doc in the `configuration.md`.

Reply:
Sure. Added.